orb-ui/src/engine/animations/slider.rs

```diff
@@ -57,6 +59,7 @@ impl<const N: usize> Slider<N> {
     }
 
     /// Enable pulsing
+    #[allow(dead_code)]
```

Comment:
The new rust version adds the `expect` lint, which will error if the lint is not triggered. We should prefer that lint, so that we get informed when the exception is no longer necessary.

https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html#expectlint